logic.js

```diff
@@ -0,0 +1,93 @@
+
+const Tweeter = function(){
+  let posts = [ {
```

Comment:
`const`

you do not need to you `let` very often. Your default should be `const`